cypress/spec/work_type_modal.cy.js

```diff
@@ -62,7 +62,7 @@ describe('Work type modal', () => {
       cy.get('div.subtype-container[data-work-type-target="subtype"] div.subtype-item input').last().click()
       cy.get('button[type="submit"]').click()
     })
-    cy.url().should('include', `/collections/${collection_id}/works/new?work_type=mixed+material&subtype%5B%5D=Data&subtype%5B%5D=Video`)
+    cy.url().should('include', `/collections/${collection_id}/works/new?work_type=mixed+material&subtype%5B%5D=3D+model&subtype%5B%5D=Working+paper`)
   })
```

Comment:
the first and last subtypes selected are slightly different now, and the test needs to reflect this